#### Test (Fail) 55858891 Build Logs


```


```

```
Already up-to-date.

From https://github.com/thaliproject/TestDummy
   9bd187f..2c5c7cb  test_jxc   -> origin/test_jxc

```

```
Your branch is up-to-date with 'origin/master'.
Branch test_ogz2 set up to track remote branch test_ogz2 from origin.
Already up-to-date.

Switched to branch 'master'
Switched to a new branch 'test_ogz2'
Note: checking out '2c5c7cb'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 2c5c7cb... Use debug version.

```

```
jxc@1.0.11 node_modules/jxc
â””â”€â”€ jxtools@0.0.3 (progress@1.1.8, adm-zip@0.4.7)
Test 0 failure
[36mDownloading:[39m [32mhttp://az836273.vo.msecnd.net/0.0.8/io.jxcore.node.jx
[39m
[1A[KAttempt 1 / 3
[1A[K[1A[K[36mProgress:[39m [32mDone[39m
Test 1 failure
[35mSkipping the download, since file io.jxcore.node.jx already exists.[39m
If you want to force downloading, use --force option.

/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:74
  jxtools.console.debugInfo('findJXandUnpack called');
                  ^
TypeError: Object #<Object> has no method 'debugInfo'
    at findJXandUnpack (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:74:19)
    at Object.exports.run (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:191:5)
    at _run (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/bin/jxc.bin.js:34:7)
    at Object.<anonymous> (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/bin/jxc.bin.js:44:3)
    at Module._compile (module.js:619:26)
    at Object.Module._extensions..js (module.js:649:10)
    at Module.load (module.js:440:36)
    at Function.Module._load (module.js:405:12)
    at Function.Module.runMain (module.js:849:12)
    at startup (node.js:485:18)
Test 2 failure
[35mSkipping the download, since file io.jxcore.node.jx already exists.[39m
If you want to force downloading, use --force option.

/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:74
  jxtools.console.debugInfo('findJXandUnpack called');
                  ^
TypeError: Object #<Object> has no method 'debugInfo'
    at findJXandUnpack (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:74:19)
    at Object.exports.run (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:191:5)
    at _run (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/bin/jxc.bin.js:34:7)
    at Object.<anonymous> (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/bin/jxc.bin.js:44:3)
    at Module._compile (module.js:619:26)
    at Object.Module._extensions..js (module.js:649:10)
    at Module.load (module.js:440:36)
    at Function.Module._load (module.js:405:12)
    at Function.Module.runMain (module.js:849:12)
    at startup (node.js:485:18)
Test 3 failure
[35mSkipping the download, since file io.jxcore.node.jx already exists.[39m
If you want to force downloading, use --force option.

/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:74
  jxtools.console.debugInfo('findJXandUnpack called');
                  ^
TypeError: Object #<Object> has no method 'debugInfo'
    at findJXandUnpack (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:74:19)
    at Object.exports.run (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:191:5)
    at _run (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/bin/jxc.bin.js:34:7)
    at Object.<anonymous> (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/bin/jxc.bin.js:44:3)
    at Module._compile (module.js:619:26)
    at Object.Module._extensions..js (module.js:649:10)
    at Module.load (module.js:440:36)
    at Function.Module._load (module.js:405:12)
    at Function.Module.runMain (module.js:849:12)
    at startup (node.js:485:18)
Test 4 failure
[35mSkipping the download, since file io.jxcore.node.jx already exists.[39m
If you want to force downloading, use --force option.

/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:74
  jxtools.console.debugInfo('findJXandUnpack called');
                  ^
TypeError: Object #<Object> has no method 'debugInfo'
    at findJXandUnpack (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:74:19)
    at Object.exports.run (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:191:5)
    at _run (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/bin/jxc.bin.js:34:7)
    at Object.<anonymous> (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/bin/jxc.bin.js:44:3)
    at Module._compile (module.js:619:26)
    at Object.Module._extensions..js (module.js:649:10)
    at Module.load (module.js:440:36)
    at Function.Module._load (module.js:405:12)
    at Function.Module.runMain (module.js:849:12)
    at startup (node.js:485:18)
Test 5 failure
[35mSkipping the download, since file io.jxcore.node.jx already exists.[39m
If you want to force downloading, use --force option.

/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:74
  jxtools.console.debugInfo('findJXandUnpack called');
                  ^
TypeError: Object #<Object> has no method 'debugInfo'
    at findJXandUnpack (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:74:19)
    at Object.exports.run (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:191:5)
    at _run (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/bin/jxc.bin.js:34:7)
    at Object.<anonymous> (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/bin/jxc.bin.js:44:3)
    at Module._compile (module.js:619:26)
    at Object.Module._extensions..js (module.js:649:10)
    at Module.load (module.js:440:36)
    at Function.Module._load (module.js:405:12)
    at Function.Module.runMain (module.js:849:12)
    at startup (node.js:485:18)
Test 6 failure
[35mSkipping the download, since file io.jxcore.node.jx already exists.[39m
If you want to force downloading, use --force option.

/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:74
  jxtools.console.debugInfo('findJXandUnpack called');
                  ^
TypeError: Object #<Object> has no method 'debugInfo'
    at findJXandUnpack (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:74:19)
    at Object.exports.run (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:191:5)
    at _run (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/bin/jxc.bin.js:34:7)
    at Object.<anonymous> (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/bin/jxc.bin.js:44:3)
    at Module._compile (module.js:619:26)
    at Object.Module._extensions..js (module.js:649:10)
    at Module.load (module.js:440:36)
    at Function.Module._load (module.js:405:12)
    at Function.Module.runMain (module.js:849:12)
    at startup (node.js:485:18)
Test 7 failure
[35mSkipping the download, since file io.jxcore.node.jx already exists.[39m
If you want to force downloading, use --force option.

/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:74
  jxtools.console.debugInfo('findJXandUnpack called');
                  ^
TypeError: Object #<Object> has no method 'debugInfo'
    at findJXandUnpack (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:74:19)
    at Object.exports.run (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:191:5)
    at _run (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/bin/jxc.bin.js:34:7)
    at Object.<anonymous> (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/bin/jxc.bin.js:44:3)
    at Module._compile (module.js:619:26)
    at Object.Module._extensions..js (module.js:649:10)
    at Module.load (module.js:440:36)
    at Function.Module._load (module.js:405:12)
    at Function.Module.runMain (module.js:849:12)
    at startup (node.js:485:18)
Test 8 failure
[35mSkipping the download, since file io.jxcore.node.jx already exists.[39m
If you want to force downloading, use --force option.

/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:74
  jxtools.console.debugInfo('findJXandUnpack called');
                  ^
TypeError: Object #<Object> has no method 'debugInfo'
    at findJXandUnpack (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:74:19)
    at Object.exports.run (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:191:5)
    at _run (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/bin/jxc.bin.js:34:7)
    at Object.<anonymous> (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/bin/jxc.bin.js:44:3)
    at Module._compile (module.js:619:26)
    at Object.Module._extensions..js (module.js:649:10)
    at Module.load (module.js:440:36)
    at Function.Module._load (module.js:405:12)
    at Function.Module.runMain (module.js:849:12)
    at startup (node.js:485:18)
Test 9 failure
[35mSkipping the download, since file io.jxcore.node.jx already exists.[39m
If you want to force downloading, use --force option.

/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:74
  jxtools.console.debugInfo('findJXandUnpack called');
                  ^
TypeError: Object #<Object> has no method 'debugInfo'
    at findJXandUnpack (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:74:19)
    at Object.exports.run (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/lib/commands/download.js:191:5)
    at _run (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/bin/jxc.bin.js:34:7)
    at Object.<anonymous> (/Users/thali/Github/testBuild/test_jxc/node_modules/jxc/bin/jxc.bin.js:44:3)
    at Module._compile (module.js:619:26)
    at Object.Module._extensions..js (module.js:649:10)
    at Module.load (module.js:440:36)
    at Function.Module._load (module.js:405:12)
    at Function.Module.runMain (module.js:849:12)
    at startup (node.js:485:18)
-e [0;31mcompilation aborted
 [0m
build aborted
 [0m

npm http GET https://registry.npmjs.org/adm-zip
npm http GET https://registry.npmjs.org/progress
npm http 200 https://registry.npmjs.org/progress
npm http GET https://registry.npmjs.org/progress/-/progress-1.1.8.tgz
npm http 200 https://registry.npmjs.org/adm-zip
npm http GET https://registry.npmjs.org/adm-zip/-/adm-zip-0.4.7.tgz
npm http 200 https://registry.npmjs.org/progress/-/progress-1.1.8.tgz
npm http 200 https://registry.npmjs.org/adm-zip/-/adm-zip-0.4.7.tgz
./build_droid.sh: line 31: EXIT_WITH_FAILURE_ON_PURPOSE: command not found

```

Error: Command failed: npm http GET https://registry.npmjs.org/adm-zip
npm http GET https://registry.npmjs.org/progress
npm http 200 https://registry.npmjs.org/progress
npm http GET https://registry.npmjs.org/progress/-/progress-1.1.8.tgz
npm http 200 https://registry.npmjs.org/adm-zip
npm http GET https://registry.npmjs.org/adm-zip/-/adm-zip-0.4.7.tgz
npm http 200 https://registry.npmjs.org/progress/-/progress-1.1.8.tgz
npm http 200 https://registry.npmjs.org/adm-zip/-/adm-zip-0.4.7.tgz
./build_droid.sh: line 31: EXIT_WITH_FAILURE_ON_PURPOSE: command not found
